library/network_connections.py

```diff
@@ -457,6 +457,10 @@ def ifcfg_create(
                 ifcfg["IPV6_AUTOCONF"] = "no"
             else:
                 ifcfg["IPV6INIT"] = "no"
+
+            if ip["ipv6_disabled"]:
+                ifcfg["IPV6_DISABLED"] = "yes"
```

Comment:
`grep -R IPV6_DISABLED /etc/sysconfig/` shows that initscripts don't support this variable. This is an extension added by NetworkManager, but when the role writes ifcfg-rh files, it must write them so that initscripts can handle them.

I don't think that initscript support `ipv6_disabled`. At least not as part of ifcfg-files, you would have to write a sysctl files (and possibly reboot). So, it's fine if the ifcfg-rh backend does not support this option. See `validate_connection_one()` how to handle that.

Reply:
these two lines are still here in the latest version.

Reply:
these two lines were removed